packages/renderer/src/lib/image/RunImage.svelte

```diff
@@ -69,68 +170,661 @@ async function startContainer() {
+                    <button
+                      class="ml-2 p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400"
+                      hidden="{index === environmentVariables.length - 1}"
+                      on:click="{() => deleteEnvVariable(index)}">
+                      <Fa class="h-4 w-4 text-xl" icon="{faMinusCircle}" />
+                    </button>
+                    <button
+                      class="ml-2 p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400"
+                      hidden="{index < environmentVariables.length - 1}"
+                      on:click="{addEnvVariable}">
+                      <Fa class="h-4 w-4 text-xl" icon="{faPlusCircle}" />
+                    </button>
+                  </div>
+                {/each}
+              </div>
+            </Route>
+            <Route path="/advanced">
+              <div class="h-96 overflow-y-auto pr-4">
+                <!-- Use tty -->
+                <label for="containerTty" class="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Use tty:</label>
+                <div class="flex flex-row justify-start items-center align-middle w-full text-gray-400 text-sm">
+                  <input type="checkbox" bind:checked="{useTty}" class="mx-2 outline-none text-sm" />
+                  Attach a pseudo terminal
+                </div>
+
+                <!-- Specify user-->
+                <label for="containerUser" class="pt-4 block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Specify user to run container as:</label>
+                <div class="flex flex-row justify-center items-center w-full">
+                  <input
+                    type="text"
+                    bind:value="{runUser}"
+                    placeholder="If you specify a name, user must exists in passwd file (use user id instead)"
+                    class="ml-2 w-full p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400" />
+                </div>
+
+                <!-- Autoremove-->
+                <label
+                  for="containerAutoRemove"
+                  class="pt-4 block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Auto Removal of containers:</label>
+                <div class="flex flex-row justify-start items-center align-middle w-full text-gray-400 text-sm">
+                  <input type="checkbox" bind:checked="{autoRemove}" class="mx-2 outline-none text-sm" />
+                  Automatically remove the container when the container's process exits
+                </div>
+
+                <!-- RestartPolicy-->
+                <label
+                  for="containerRestartPolicy"
+                  class="pt-4 block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Restart Policy. Behaviour when container is exiting:</label>
```

Comment:
```suggestion
                  >Restart policy:</label>
```

I feel like this could be shortened to just `Restart policy:`